src/traced/probes/ftrace/ftrace_config_muxer.cc

```diff
@@ -662,13 +351,15 @@ FtraceConfigMuxer::FtraceConfigMuxer(
     ProtoTranslationTable* table,
     SyscallTable syscalls,
     std::map<std::string, std::vector<GroupAndName>> vendor_events,
+    std::map<std::string, std::set<GroupAndName>> predefined_events,
```

Comment:
Can we use base::FlatSet<GroupAndName> instead of std::set<GroupAndName> (here and everywhere), given that we expect a small number of events per category?